### ISSUE TITLE
Clarification: Section 13.4.4 - p. 553 - Shared Offset

### ISSUE THREAD

**Issue by wesbland** [mpi-forum/mpi-issues#267]
## Problem

Is "shared offset" really "offset implied by the shared file pointer"? See, e.g., the paragraph after "Collective Operations" after MPI_FILE_IWRITE_SHARED ("shared offce accesses by the same …"

### Suggested Fix

Change "shared offset" to "offset implied by the shared file pointer"

**Comment by wesbland** [comment]
The chapter committee can propose text, but maybe this is an errata:

* Editor - 1
* Chapter committee - 4
* Errata - 2
* Full Proposal - 0

**Comment by rsth** [comment]
Yes, but if you insert the suggested change into the sentence, it sounds worse than before. 

Existing sentence: "in order to prevent subsequent shared offset accesses by the same processes from interfering with this collective access"

With the suggested change: "in order to prevent subsequent offset implied by the shared file pointer accesses by the same processes from interfering with this collective access"

Maybe leave it as is because the reader will understand what it means?

**Comment by tonyskjellum** [comment]
@wesbland this will be addressed for MPI-4.0

**Comment by Shinji-Sumimoto** [comment]
This is now located on page 674 L40 in mpi40-report-RC1.pdf.
